electron_app/src/webcontents-handler.js

```diff
@@ -84,14 +84,6 @@ function _CutCopyPasteSelectContextMenus(params) {
     }];
 }
 
-function onSelectedContextMenu(ev, params) {
```

Comment:
my guess is the logic needs inverting and you need to override the input menu, but leave the highlighted text menu